templates/repo/actions/runs_list.tmpl

```diff
@@ -1,14 +1,18 @@
 <div class="issue list">
 	{{range .Runs}}
-		<li class="item gt-df gt-py-3">
+		<li class="item gt-df gt-py-3 gt-ab">
 			<div class="issue-item-left gt-df">
 				{{template "repo/actions/status" .Status}}
 			</div>
-			<div class="issue-item-main gt-f1 gt-fc gt-df">
+			<div class="issue-item-main action-issue-item gt-f1 gt-fc gt-df gt-mr-3">
 				<div class="issue-item-top-row">
-					<a class="index gt-ml-0 gt-mr-2" href="{{if .Link}}{{.Link}}{{else}}{{$.Link}}/{{.Index}}{{end}}">
+					<a class="index gt-ml-0 gt-mr-2" title={{.Title}} href="{{if .Link}}{{.Link}}{{else}}{{$.Link}}/{{.Index}}{{end}}">
```

Comment:
```suggestion
					<a class="index gt-ml-0 gt-mr-2" title="{{.Title}}" href="{{if .Link}}{{.Link}}{{else}}{{$.Link}}/{{.Index}}{{end}}">
```

Reply:
And one more thing, the font seems too large in the "issue-item-top-row", the font size is suitable to be used as "Issue Title", but it's not suitable for "Commit Message"